libs/wingsdk/src/target-sim/bucket.ts

```diff
@@ -41,6 +42,15 @@ export class Bucket extends cloud.Bucket implements ISimulatorResource {
     this.initialObjects[key] = body;
   }
 
+  public addFile(fileName: string, path: string): void {
+    path = isAbsolute(path)
+      ? path
+      : resolve(process.env.WING_SOURCE_DIR ?? "", path);
+    const data = fs.readFileSync(path, File.getFileEncoding(path));
```

Comment:
since this code block is the same in all of the target, we can add it to the parent class :) 
here: libs/wingsdk/src/cloud/bucket.ts